### DISCUSSION TITLE
No module named '_curses' on CentOS7

### DISCUSSION THREAD

**Discussion by ransay22** [ericaltendorf/plotman#948] [Q&A]
Please help on how to resolve this issue. I already try installing the curses manually but still it wont work, error goes like the following:

`(venv) [tractor0@localhost chia-blockchain]$ plotman config generate`
```
Traceback (most recent call last):
  File "/home/tractor0/venv/bin/plotman", line 5, in <module>
    from plotman.plotman import main
  File "/home/tractor0/venv/lib/python3.7/site-packages/plotman/plotman.py", line 18, in <module>
    from plotman import (
  File "/home/tractor0/venv/lib/python3.7/site-packages/plotman/interactive.py", line 1, in <module>
    import curses
  File "/usr/local/lib/python3.7/curses/__init__.py", line 13, in <module>
    from _curses import *
ModuleNotFoundError: No module named '_curses'
```

Hope someone has an idea on how to resolve this issue. it greatly helps.

Thanks.

Richard

**Comment by altendky** [comment]
If you search the issues (as opposed to discussions) there are a number of them discussing this.  I'm not sure which exactly will apply for your case.  How did you install python?

**Reply by ransay22** [reply]
i think python was installed together with the Chia thing, thanks for the info i will try to search further , but upon initial search they are using like centos 6 and centos8 i was hoping that someone, have same OS version that i have and have experiences on this kind of issue. thanks once again.

**Comment by ransay22** [comment]
issues resolve i guess, it don't give me error now, i just recompile the python and reinstall it. thanks, i read it on the other post with curses issue
